pages/index.tsx

```diff
@@ -1,15 +1,32 @@
-import { useEffect, useState } from 'react';
-
 import Trending from '../components/trending/Trending';
 import request from '../modules/request';
-import { AppWrapper, GlobalStyle } from '../styles/index.styles';
+import { AppWrapper, GlobalStyle, Navigation } from '../styles/index.styles';
+
+interface Community {
```

Comment:
You changed this in the other PR. That's probably why this PR has conflicts with main branch and cant be merged. Plz merge main into this branch and fix the conflicts.

Reply:
Done!

Reply:
Done!